api/swagger.yaml

```diff
@@ -5668,10 +5668,16 @@ paths:
 
         Volumes report these events: `create, mount, unmount, destroy`
 
-        Networks report these events: `create, connect, disconnect, destroy`
+        Networks report these events: `create, connect, disconnect, destroy, update, remove`
```

Comment:
oh, actually, can you change these to:

    `create`, `connect`, `disconnect`, `destroy`, `update`, and `remove`

(each option in its own "code" block, instead of all in one code-block)